openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -38,59 +41,9 @@ def test_get_programs(self):
         self.assertEqual(len(httpretty.httpretty.latest_requests), 1)
 
     @httpretty.activate
-    def test_get_programs_caching(self):
```

Comment:
Why are these tests being deleted? The tests added to cover the new api_utils module aren't as extensive (e.g., they don't check that the cache is bypassed for staff users). Leaving these in place will help verify that your changes here haven't broken existing programs functionality.